src/test/unittests.sh

```diff
@@ -5,14 +5,14 @@
 err=0
 trap 'err=1' ERR
 
-python3 ../hammer-vlsi/test.py
-python3 ../hammer-vlsi/tech_test.py
-python3 ../hammer-vlsi/constraints_test.py
-python3 ../hammer-vlsi/cli_driver_test.py
-python3 ../hammer-vlsi/utils_test.py
-python3 ../hammer-vlsi/units_test.py
-python3 ../hammer-vlsi/verilog_utils_test.py
-python3 ../hammer-vlsi/lef_utils_test.py
-python3 ../hammer_config_test/test.py
+python3 -m pytest ../hammer-vlsi/vlsi_test.py
```

Comment:
Minor nitpick, can you set up the test files so they can still be run via `python3 ../hammer-vlsi/vlsi_test.py`?

Reply:
OK, I'll fix this, and `pytest` will be still work in single command `pytest` too.